resources/assets/components/InboxItem/index.js

```diff
@@ -52,12 +70,12 @@ class InboxItem extends React.Component {
           </ul>
         </div>
         <div className="container__block -third">
-          {post['user'] ?
+          {post['user'] || signup.user ?
             <div>
-              <h2>{post['user']['first_name']} {post['user']['last_name']}, {calculateAge(post['user']['birthdate'])}</h2>
+              <h2>{first_name} {last_name}, {birthdate}</h2>
```

Comment:
It should already be available on the user response that's returned from Gateway in each case, but just not automatically bundled when using `toArray()`. So you should be able to attach it as another field in the response [here](https://github.com/DoSomething/rogue/blob/ac76c91a7607afd55adee9f523270ee3c3cc9561/app/Http/Controllers/CampaignsController.php#L75) and [here](https://github.com/DoSomething/rogue/blob/ac76c91a7607afd55adee9f523270ee3c3cc9561/app/Http/Controllers/CampaignsController.php#L108):

```diff
- $item->user = $user->toArray() ?: ['first_name' => 'A Doer'];
+ $item->user = [
+   'displayName' => $user->displayName(),
+   'birthdate' => $user->birthdate,
+   // other fields you want to include...
+ ];
```